user_docs/md2html.py

```
@@ -3,16 +3,14 @@
 # This file may be used under the terms of the GNU General Public License, version 2 or later.
 # For more details see: https://www.gnu.org/licenses/gpl-2.0.html
 
+import sys
```

Ruff

`sys` imported but unused

Remove unused import: `sys`

(F401)